shardeum/network-status
# Add support for docker image

This pull request introduces Docker configuration files to integrate the backend and frontend services of the network-status application into a Docker environment. Dockerfiles are provided for both backend and frontend services, along with a `docker-compose.yml` file to define and run the Docker containers for both services. This allows for easier deployment and management of the application in various environments.

## Testing Instructions

Make sure you have `docker` and `docker-compose` installed, follow the [official documentation ](https://docs.docker.com/get-docker/).

To run the Docker image, execute the following commands: 

```
docker-compose build
docker-compose up
```

After running these commands, the application should be accessible via the configured ports, providing a seamless Docker-based environment for development and deployment.

<img width="1440" alt="Screenshot 2024-06-07 at 20 25 25" src="https://github.com/shardeum/network-status/assets/33036742/efd711e2-21ef-4acf-81c0-8d43af08d081">
<img width="1440" alt="Screenshot 2024-06-07 at 20 24 58" src="https://github.com/shardeum/network-status/assets/33036742/5cb211cd-57ef-4bca-8d7c-09571aa95c24">
<img width="1440" alt="Screenshot 2024-06-07 at 20 25 07" src="https://github.com/shardeum/network-status/assets/33036742/ce49567d-99aa-4579-9f2e-f90111f1b6ab">
